docs/rancher/07-run-murmuration-services/README.md

```diff
@@ -0,0 +1,164 @@
+protoprod-index.murmurations.network       -->     my-index.murmurations.network
+protoprod-library.murmurations.network     -->     my-library.murmurations.network
+protoprod-data-proxy.murmurations.network  -->     my-data-proxy.murmurations.network
+```
+
+## Step 5 - Creating Required Secrets
+
+Create Kubernetes secrets for MongoDB credentials and any other necessary secrets for the operation of Murmurations services:
+
+Remember to replace those "password" with strong ones.
+
+```bash
+kubectl \
+  create secret generic index-mongo-secret \
+  --from-literal="MONGO_INITDB_ROOT_USERNAME=index-admin" \
+  --from-literal="MONGO_INITDB_ROOT_PASSWORD=password"
+
+kubectl \
+  create secret generic library-mongo-secret \
+  --from-literal="MONGO_INITDB_ROOT_USERNAME=library-admin" \
+  --from-literal="MONGO_INITDB_ROOT_PASSWORD=password"
+
+kubectl \
+  create secret generic data-proxy-mongo-secret \
+  --from-literal="MONGO_INITDB_ROOT_USERNAME=data-proxy-admin" \
+  --from-literal="MONGO_INITDB_ROOT_PASSWORD=password"
+
+kubectl \
+  create secret generic index-secret \
+  --from-literal="MONGO_USERNAME=index-admin" \
+  --from-literal="MONGO_PASSWORD=password"
+
+kubectl \
+  create secret generic library-secret \
+  --from-literal="MONGO_USERNAME=library-admin" \
+  --from-literal="MONGO_PASSWORD=password"
+
+kubectl \
+  create secret generic data-proxy-secret \
+  --from-literal="MONGO_USERNAME=data-proxy-admin" \
+  --from-literal="MONGO_PASSWORD=password"
+
+kubectl \
+  create secret generic nodecleaner-secret \
+  --from-literal="MONGO_USERNAME=index-admin" \
+  --from-literal="MONGO_PASSWORD=password"
+
+kubectl \
+  create secret generic revalidatenode-secret \
+  --from-literal="MONGO_USERNAME=index-admin" \
+  --from-literal="MONGO_PASSWORD=password"
+
+kubectl \
+  create secret generic dataproxyupdater-secret \
+  --from-literal="MONGO_USERNAME=data-proxy-admin" \
+  --from-literal="MONGO_PASSWORD=password"
+
+kubectl \
+  create secret generic dataproxyrefresher-secret \
+  --from-literal="MONGO_USERNAME=data-proxy-admin" \
+  --from-literal="MONGO_PASSWORD=password"
+
+kubectl \
+  create secret generic schemaparser-secret \
+  --from-literal="MONGO_USERNAME=library-admin" \
+  --from-literal="MONGO_PASSWORD=password" \
+  --from-literal="GITHUB_TOKEN=<GITHUB_TOKEN>"
+```
+
+For how to obtain the `GITHUB_TOKEN`, please refer to [GitHub's documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for creating a personal access token. No additional scopes are needed.
+
+## Step 6 - Deploying the Services
+
+Deploy the Murmurations services to your Kubernetes cluster:
+
+```bash
+make deploy-all-services DEPLOY_ENV=<environment>
+```
+
+Note we currently have 4 environments: production, staging, pretest, and development. If in step 4 you updated the production URL, then you can use `make deploy-all-services DEPLOY_ENV=production`.
```

Comment:
I believe that the production environment difference includes more than just a URL. It includes distinct settings for cron jobs as well. I suggest creating a table here to inform users about the specific environment they wish to deploy.

Reply:
Do you think this is ok? 
https://github.com/MurmurationsNetwork/MurmurationsServices/blob/e6e29a3797ca67fccbe4ecd98fa1b41d1d5b33b1/docs/rancher/07-run-murmuration-services/README.md#step-4---updating-ingress-configuration